src/collection-browser.ts

```diff
@@ -136,6 +136,8 @@ export class CollectionBrowser
 
   @property({ type: Boolean }) suppressSortBar = false;
 
+  @property({ type: Boolean }) suppressDisplayModes = false;
```

Comment:
idk how I feel about a new property, can we just do it in the resize callback and save us a new property at the top level?

Reply:
Unfortunately the use case here isn't tied to a specific sort bar width like the sort options. We want these display modes to disappear when a patron has expanded the search bar on the profile page, and re-appear when they focus elsewhere, regardless of the width of the page or sort bar container.

I'm no fan of adding new properties, but I think in this case we do need one for the sake of passing UI state. Perhaps a later refactoring pass should attempt to consolidate some of our various boolean properties into a single options object.